### PR TITLE
[FLINK-7570] Fix broken link on faq page

### PR DESCRIPTION
**Pull request by yew1eb**
[https://issues.apache.org/jira/browse/FLINK-7570](https://issues.apache.org/jira/browse/FLINK-7570)

### CODE REVIEW

File: content/faq.html [hunk 1]
```diff
@@ -224,7 +224,7 @@ <h3 id="do-i-have-to-install-apache-hadoop-to-use-flink">Do I have to install Ap
 Hadoop client libraries by default.</p>
 
 <p>Additionally, we provide a special YARN Enabled download of Flink for
-users with an existing Hadoop YARN cluster. <a href="http://hadoop.apache.org/docs/r2.2.0/hadoop-yarn/hadoop-yarn-site/YARN.html">Apache Hadoop
+users with an existing Hadoop YARN cluster. <a href="https://hadoop.apache.org/docs/r2.7.2/hadoop-yarn/hadoop-yarn-site/YARN.html">Apache Hadoop
```
Comment (zentol) [inline content/faq.html]: I would suggest to link to the latest stable docs instead, i.e. `http://hadoop.apache.org/docs/stable/hadoop-yarn/hadoop-yarn-site/index.html`

Comment (yew1eb) [reply content/faq.html]: Thanks @dawidwys for the suggestion. I updated the PR accordingly.

**Comment by zentol** [comment]
merging.

**Comment by yew1eb** [comment]
Hi @zentol,
I suggest setting the default branch of this repository to `asf-site`.
Best,
Hai Zhou

**Pull request closed without merging**